### DISCUSSION TITLE
Implemented iRetryAnalyzer in testng framework to retry failed test cases once and executing only two test cases, however in the report it is showing 3 test cases

### DISCUSSION THREAD

**Discussion by reddypramodhadalam** [community/community#114372] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

<img width="960" alt="ExtentReport" src="https://github.com/community/community/assets/33780201/9eaeb6a2-6284-46fd-b563-8044471a31fa">

here failed test case is retried, however it is showing as passed, Below is the maven logs.
[ERROR] Tests run: 2, Failures: 1, Errors: 0, Skipped: 0
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  11.308 s
[INFO] Finished at: 2024-03-26T16:04:22+05:30
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.2.5:test (default-test) on project Demo: There are test failures.

issue is in the extent html report it is showing 3 test cases even though the total number of test cases executing is only 2

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @reddypramodhadalam! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, you are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.

I've gone ahead and moved this to the correct category for you. Good luck!
